tests/yam/agama/agama_integration.pm

```diff
@@ -1,8 +1,8 @@
-## Copyright 2023 SUSE LLC
+## Copyright 2024 SUSE LLC
 # SPDX-License-Identifier: GPL-2.0-or-later
 
-# Summary: Run interactive installation with Agama,
-# run playwright tests directly from the Live ISO.
+# Summary: Run automated installation with Agama,
+# run puppeteer tests directly from the Live ISO.
```

Comment:
s/run puppeteer/using a web automation tool/

Let's not use puppeteer or specific framework for naming things or even comments, we do functional testing (our test should be about funcionality, the HOW can be hidden most of the time. For what I could see last time with playwright we were too specific with this and names degenerated, so yeah, better address this from begining.

Reply:
Solved aca47525aabf6b10f5285e7a04d1d88eee7da4d4